src/Admin/AbstractAdmin.php

```diff
@@ -2355,57 +2244,47 @@ public function setPerPageOptions(array $options): void
 
     /**
      * Returns predefined per page options.
-     *
-     * @return array
      */
-    public function getPerPageOptions()
+    public function getPerPageOptions(): array
     {
         return $this->perPageOptions;
     }
 
     /**
      * Set pager type.
-     *
-     * @param string $pagerType
      */
-    public function setPagerType($pagerType): void
+    public function setPagerType(string $pagerType): void
     {
         $this->pagerType = $pagerType;
     }
 
     /**
      * Get pager type.
-     *
-     * @return string
      */
-    public function getPagerType()
+    public function getPagerType(): string
     {
         return $this->pagerType;
     }
 
     /**
      * Returns true if the per page value is allowed, false otherwise.
-     *
-     * @param int $perPage
-     *
-     * @return bool
      */
-    public function determinedPerPageValue($perPage)
+    public function determinedPerPageValue(int $perPage): bool
     {
         return \in_array($perPage, $this->perPageOptions, true);
     }
 
-    public function isAclEnabled()
+    public function isAclEnabled(): bool
     {
         return $this->getSecurityHandler() instanceof AclSecurityHandlerInterface;
     }
 
-    public function getObjectMetadata($object)
+    public function getObjectMetadata($object): MetadataInterface
```

Comment:
```suggestion
    public function getObjectMetadata(object $object): MetadataInterface
```
According to the interface